cmd/operator-sdk/build/cmd.go

```diff
@@ -61,6 +62,8 @@ For example:
 		"Tool to build OCI images. One of: [docker, podman, buildah]")
 	buildCmd.Flags().StringVar(&goBuildArgs, "go-build-args", "",
 		"Extra Go build arguments as one string such as \"-ldflags -X=main.xyz=abc\"")
+	buildCmd.Flags().BoolVar(&buildImage, "build-image", true,
```

Comment:
Since we have already the command `operator-sdk build` which will gen the Dockerfile and image then, WDYT about the flag is `--skip-image` since this is your goal? 

So, w can simplify it by if has the flag than the image will not b built.

Reply:
Sounds good.

Reply:
Ok, I had misunderstood your intentions here, I thought you meant that if we discover the user hasn't asked for an image by specifying the name at all, we just don't make an image. That's what I had implemented. I'll redo it with the flag.

Reply:
Sorry @joibel, could explain better first as well my idea/suggestion.
really thank you for your contribution.